src/components/ui/form-elements.js

```diff
@@ -105,19 +104,11 @@ export const Textarea = ({
       />
 
       <textarea
-        className={
-          `${
-            errorMessage ? 'border-red-500 text-red-500 ' : null
-          } px-4 py-3 rounded-lg text-sm font-normal bg-gray-50 h-28 border border-solid border-gray-300 w-full focus:border-gray-900 focus:ring-transparent focus:outline-none mt-2 ` +
-          className
-        }
+        className={` h-28 ${fieldStyles} ${className}`}
```

Comment:
```suggestion
        className={`h-28 ${fieldStyles} ${className}`}
```
Not sure if prettier suggested the space, if it did, keep the space.